AI Guidelines/iOS/Localize - Sentiary.md

```
@@ -0,0 +1,62 @@
+
+### c. Examples
+-   **Good:** `L10n.settings_profile_save_button_label`
+    -   *Location:* Settings
+    -   *View:* Profile
+    -   *Component:* Save Button
+    -   *Property:* Label
+-   **Bad:** `L10n.save_my_changes_to_profile` (Based on content, not location/function)
+
+---
+
+## 3. **Upload Strings to Sentiary**
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)